todo_app/app.py

```diff
@@ -68,15 +72,14 @@ class Lists():
     list_types_forward = {"To Do":"todo","Doing":"doing","Done":"done"}
     list_types_reverse = {"todo":"To Do","doing":"Doing","done":"Done"}
     def __init__(self,listtype):
-        boardname = Config().BOARD_NAME
-        trelloboard = get_BoardId(boardname)
-        if not trelloboard:
-            raise ValueError(boardname+": Board not found, check your Trello account/Board name")
+        db_name = Config().DEFAULT_DATABASE
+        mongo_db = get_mongo_db(db_name)
+        if not mongo_db:
+            raise ValueError(db_name+": MongoDB not found, check your setup")
             exit()
-       
         self.listName = listtype
         self.listKey = self.list_types_forward[self.listName]
-        self.listID = find_list(trelloboard,self.listName)
+        self.listID = mongo_db[(find_list(mongo_db,self.listName))]
```

Comment:
Mentioned below, but if you switch this to the below then your app will gracefully handle the case that the list doesn't yet exist, so there's no need to first validate the list does exist
```suggestion
        self.listID = mongo_db[mongo_db,self.listName]
```

Also worth noting that the returned object is a collection object not just the list's id, so the name is a little confusing